build/index.js

```diff
@@ -259,7 +259,6 @@ function getOptions (dev) {
     indexable: indexable,
     unmergeable: ['examples', 'description', 'extra'],
     filenameModifier: filenameModifier,
-    outputLatest: false,
```

Comment:
Not sure why this was false but it was preventing files from being created that ended up solving the whole issue